### ISSUE TITLE
New libav breaks support for vdpau acceleration in libavg

### ISSUE THREAD

**Issue by uzadow** [libavg/libavg#620]
(Creating a new issue for this - I hijacked the old one by mistake)

With libav in Ubuntu 14.10 (libavcodec version 56.1.0), libavg vdpau support no longer works:
```
FAIL: testVideoAccel (AVTest.AVTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/uzadow/devel/libavg/libavg/src/test/AVTest.py", line 656, in testVideoAccel
    self.assertEqual(video.accelerated, (accelConfig != avg.NO_ACCELERATION))
AssertionError: False != True
```
The failing line is in VDPAUDecoder::openCodec:
```
pCodec = avcodec_find_decoder_by_name("mpeg1video_vdpau");
```

**Comment by uzadow** [comment]
I got confirmation from one of the libav developers that the interface we've been using for VDPAU acceleration has been removed completely from libav and ffmpeg and replaced by a new interface. Here's his mail: https://lists.libav.org/pipermail/libav-tools/2015-March/000550.html

This means that if we want hardware-accelerated video on NVidia machines, someone has to rewrite the complete support :-/.

**Comment by uzadow** [comment]
From @klemmster: 
Maybe it would be most useful to revive https://github.com/libavg/libavg/tree/libavg_vaapi,
as this would cover nvidia, amd and intel instead of only nvidia. (at least according to ArchlinuxWiki)

**Comment by uzadow** [comment]
I just checked with current ffmpeg (ver 2.6, not libav): Everything is fine there. Also, ffmpeg will be included in Ubuntu starting from 15.04 again, so I think we can let the issue rest until then and switch over to ffmpeg completely :-).
